README.md

```diff
@@ -17,11 +17,11 @@ country risk is a more general term that generally refers only to risks affectin
 ## Container images
 
 This application provides container images for demonstration purposes.
-The base image used, to build this demo application image is `node:18-alpine` and `nginxinc/nginx-unprivileged:stable-alpine`
+The base image used, to build this demo application image is `node:20-alpine` and `nginxinc/nginx-unprivileged:alpine`
```

Comment:
I think you should introduce separate, dedicated notice for docker image files. Right now you have two sections in your top-level README about images and the one dedicated file only contains info about the frontend base image.
I'll create an issue for it describing the details. If I missed something, we can close it right away

Reply:
Issue created: #45